src/content/reference/app-configuration/index.md

```diff
@@ -160,52 +155,48 @@ data:
+templated Secret, then you have to base64 encode it to comply with how Kubernetes stores secrets. Our API only encodes the entire secrets value string. It does not individually encode each value.
 
-_Our API only encodes the entire secrets value string. It does not individually encode each
-value_
+## How configuration values are stored and referenced in the Control Plane {#storage-referencing}
 
-
-## How are configuration values stored and referenced in the Control Plane?
-
-Configuration for `Apps` are stored as `ConfigMaps` and `Secrets`, which are
-referenced by `name` and `namespace` in various `spec` fields of the `App` and `AppCatalog` Custom Resource.
+Configuration for Apps are stored as ConfigMaps and Secrets, which are
+referenced by `name` and `namespace` in various `spec` fields of the [App](/reference/cp-k8s-api/apps.application.giantswarm.io/) and [AppCatalog](/reference/cp-k8s-api/appcatalogs.application.giantswarm.io/) Custom Resource (CR).
 
 Our operators act on those resources to ensure the actual state ends up
-looking like the desired state. More information is available in a [general overview of the App Catalog](/basics/app-catalog/).
+looking like the desired state. More information is available in our [general overview of the App Catalog](/basics/app-catalog/).
 
 |Configuration Level|Where to set it|Fields to set|
 |---|---|---|
-|`catalog`|`Catalog CR`|`spec.config.configMap.name`|
-|||`spec.config.configMap.namespace`|
-|||`spec.config.secret.name`|
-|||`spec.config.secret.namespace`|
+|`catalog`|`Catalog CR`|`.spec.config.configMap.name`|
+|||`.spec.config.configMap.namespace`|
+|||`.spec.config.secret.name`|
+|||`.spec.config.secret.namespace`|
 |-|-|-|
-|`app`|`App CR`|`spec.config.configMap.name`|
-|||`spec.config.configMap.namespace`|
-|||`spec.config.secret.name`|
-|||`spec.config.secret.namespace`|
+|`app`|`App CR`|`.spec.config.configMap.name`|
+|||`.spec.config.configMap.namespace`|
+|||`.spec.config.secret.name`|
+|||`.spec.config.secret.namespace`|
 |-|-|-|
-|`user`|`App CR`|`spec.userConfig.configMap.name`|
-|||`spec.userConfig.configMap.namespace`|
-|||`spec.userConfig.secret.name`|
-|||`spec.userConfig.secret.namespace`|
+|`user`|`App CR`|`.spec.userConfig.configMap.name`|
+|||`.spec.userConfig.configMap.namespace`|
+|||`.spec.userConfig.secret.name`|
+|||`.spec.userConfig.secret.namespace`|
 
-When setting user level configuration using the Giant Swarm API or our Web Interface,
-the fields in the `App CR` are edited automatically for you while creating
+When setting user level configuration using the Giant Swarm API or our [web interface](/reference/web-interface/),
+the fields in the App CR are edited automatically for you while creating
 the `ConfigMap` or `Secret`.
 
-## What is the right format for values in the ConfigMap and Secret?
+## Encoding values in ConfigMap and Secret {#encoding}
```

Comment:
Not sure about the heading having the word 'encoding' in it.

The values for secrets have to be encoded, but that is almost kubernetes storage detail.

The key under which we expect your configuration (`.data.values`) is not an encoding.

Reply:
OK, changed